client/blocks/inline-help/popover-content.scss

```diff
@@ -0,0 +1,168 @@
+.inline-help__popover-content {
+	background-color: var( --color-surface );
+
+	p {
```

Comment:
Noting that these `p` and `a, button` styles were previously applied to `.inline-help__mobile-overlay`. Should we preserve that?

Reply:
No, they were applied to `.inline-help__popover.popover`. But the GitHub diff viewer collapses the unchanged parts of `style.scss` so that it appears they were inside `.inline-help__mobile-overlay`. Please have one more look.